packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -556,6 +556,37 @@ describe('ComboBox', () => {
     expect(updatedText).toEqual('');
   });
 
+  it('Can clear text in controlled case with autoComplete off and allowFreeform on', () => {

```

Comment:
nit: this feels like a valuable test case, but a different concern from the screenreader issue.

Reply:
It is, it's making sure that I don't regress the change that originally caused the placeholder issue.